pysrc/pip_resolve.py

```diff
@@ -146,20 +181,22 @@ def satisfies_python_requirement(parsed_operator, py_version_str):
         '!=': ne,
     }[parsed_operator]
     system_py_version_tuple = (sys.version_info[0], sys.version_info[1])
-    py_version_tuple = tuple(py_version_str.split('.')) # string tuple
+    py_version_tuple = tuple(py_version_str.split('.'))  # string tuple
```

Comment:
We can consider adding some `black` or the like opinionated formatter to linting phase. Like https://github.com/snyk/engprod-webhooks-server/blob/8a6bbdf342a7b998b735621d5630b8cbb48e2cda/Makefile#L9-L15

Reply:
The holy trinity for linting in Python is usually Black + Pylint + Flake8. But I think we should add this in another PR or when refactoring the pysrc code, otherwise we will have all kind of unexpected changes, plus it will make a lot of changes and we won't be able to tell what was really implemented in this PR.

Reply:
Totally.